rskj-core/src/main/java/co/rsk/db/migration/OrchidToUnitrieMigrator.java

```diff
@@ -303,6 +297,24 @@ private void migrateContract(RskAddress accountAddress, Repository currentReposi
         }
     }
 
+    private Trie getContractStorageTrie(RskAddress contractAddress, ContractData contractData, Keccak256 oldAccountStateRoot) {
+        try {
+            Trie trie = getContractStorageTrie(contractAddress, contractData);
+            TrieStore trieStore = trie.getStore();
+            if (Arrays.equals(EMPTY_TRIE_HASH, oldAccountStateRoot.getBytes())) {
```

Comment:
I think this has been checked in `co.rsk.db.migration.OrchidToUnitrieMigrator#migrateContract`, isn't it?